tests/ncbi/test_client.py

```diff
@@ -119,7 +119,7 @@ def test_ok(self):
             "JQ821386.1",
         ]
 
-    def test_non_existent(self):
+    def test_nonexistent_fail(self):
```

Comment:
This isn't a failure. The previous naming was good.

Side note: The `test_ok` convention is just a name we use cross-project for tests of the base case in class-based tests. No need to suffix with `ok` and `fail` on other test cases.